numbers/guides/hostedMessaging.md

```diff
@@ -359,6 +378,30 @@ Messaging route of External Third Party TNs is not configured.
 5554443333
 ```
 
+{% sample lang="js" %}
+
+```js
+const numbersToCheck = ["5554443333"];
+
+try {
+  const importResults = await numbers.ImportTnChecker.checkAsync(numbersToCheck);
+  console.log(importResults.importTnCheckerPayload.importTnErrors.importTnError.code);
+  console.log(importResults.importTnCheckerPayload.importTnErrors.importTnError.description);
+  console.log(importResults.importTnCheckerPayload.importTnErrors.importTnError.telephoneNumbers.telephoneNumber);
```

Comment:
I thik the Payload is only a field on 200s. I don't see a payload in the api docs for 400 
![image](https://user-images.githubusercontent.com/45304746/88959805-953a0880-d270-11ea-89d1-ee81a56fbfd2.png)
Unless I'm looking at a different one?

Reply:
Speaking of which, I wonder if it'd be better to print the full body of the object, but that's really an aside

Reply:
It's the /importTnChecker endpoint

```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ImportTnCheckerResponse>
    <ImportTnCheckerPayload>
        <TelephoneNumbers>
            <TelephoneNumber>3032281000</TelephoneNumber>
        </TelephoneNumbers>
        <ImportTnErrors>
            <ImportTnError>
                <Code>19006</Code>
                <Description>Bandwidth numbers cannot be imported by this account at this time.</Description>
                <TelephoneNumbers>
                    <TelephoneNumber>4109235436</TelephoneNumber>
                    <TelephoneNumber>4104685864</TelephoneNumber>
                </TelephoneNumbers>
            </ImportTnError>
        </ImportTnErrors>
    </ImportTnCheckerPayload>
</ImportTnCheckerResponse>
```

Yeah it's kinda strange

Printing the full body is hit or miss in some languages so we kinda defaulted to just grabbing some important fields as the exampels

Reply:
Oh wait is this for the 200 response? Okay you're good I was looking at the 400 response. I'm guessing that's taken care of/documented elsewhere?

Reply:
Yeah the response has "errors" but not like http errors so that could be confusing too